search/src/Components/upload/uploader_category_mapper.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import cloneDeep from 'lodash/cloneDeep';
+
+
+const useStyles = (theme) => ({
+    summary: {
+        marginLeft: '2.5em'
+    },
+    row: {
+        display: 'flex',
+        alignItems: 'baseline'
+    },
+    formControl: {
+        margin: theme.spacing(1),
+        minWidth: 120,
+    },
+    span: {
+        marginRight: '1em'
```

Comment:
I think minWidth might better serve the purpose... But so would a table or another grid-based layout.